### ISSUE TITLE
Application is unresponsive

### ISSUE THREAD

**Issue by Diamondemon** [th-ch/youtube-music#714]
Hello,
I have been having a strange problem recently. I cannot launch the app anymore. When launching the app stays with a black screen then a popup appears, saying this:
![image](https://user-images.githubusercontent.com/75697279/167261222-b7d13804-b851-4cb4-9317-be7e588ab1e6.png)

And when I launch it from the terminal, here are the errors:
```console
$ youtube-music 
[115917:0507/172713.790491:ERROR:vaapi_wrapper.cc(1136)] vaQuerySurfaceAttributes failed, VA error: invalid parameter
[115917:0507/172713.790671:ERROR:vaapi_wrapper.cc(1083)] FillProfileInfo_Locked failed for va_profile VAProfileH264Main and entrypoint VAEntrypointVLD
[115917:0507/172713.790771:ERROR:vaapi_wrapper.cc(1136)] vaQuerySurfaceAttributes failed, VA error: invalid parameter
[115917:0507/172713.790864:ERROR:vaapi_wrapper.cc(1083)] FillProfileInfo_Locked failed for va_profile VAProfileH264High and entrypoint VAEntrypointVLD
[115884:0507/172714.017400:ERROR:browser_main_loop.cc(269)] Gdk: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
/usr/bin/../lib64/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../include/c++/11.2.0/bits/unique_ptr.h:659: typename std::add_lvalue_reference<element_type>::type std::unique_ptr<blink::CSSSelector []>::operator[](std::size_t) const [_Tp = blink::CSSSelector [], _Dp = std::default_delete<blink::CSSSelector []>]: Assertion 'get() != pointer()' failed.
APPIMAGE env is not defined, current application is not an AppImage
```

I am currently on Manjaro 5.17 but I don't think it matters.

I hope you can do something, I heavily rely on your app when I am working ^^'

**Comment by Araxeus** [comment]
are you using the latest version? (https://github.com/th-ch/youtube-music/releases/tag/v1.16.0)

if you do, you could try deleting `~/.config/Youtube Music/config.json` (it might be corrupted somehow)

you might want to back it up first if you don't want to lose all your settings

(if it does fix the problem, you can send here the content of your backed-up invalid config.json so we can figure out what happened)

**Comment by Diamondemon** [comment]
I installed it from the AUR so I do not know if it is the latest. It is numbered 1.16.0-2. I will check for the config file though.

**Comment by Diamondemon** [comment]
It is even worse now, I don't even have the black screen anymore, only the popup.

**Comment by Araxeus** [comment]
can you send paste here both old and new package.json?

inside a format box like:
```P4
‎```json

‎```
```

**Comment by Diamondemon** [comment]
Where should I find the `package.json` file? Don't you mean the `config.json` file?
In case it is the latter, here you go:

The old
```json
{
	"window-size": {
		"width": 1440,
		"height": 1011
	},
	"url": "https://music.youtube.com/watch?v=Ryoxb6GnRVA&list=RDAMPLOLAK5uy_mRdRE018pOvhNEkNXSYWnKSsrurdhpfXs&index=0",
	"options": {
		"tray": true,
		"appVisible": false,
		"autoUpdates": true,
		"hideMenu": false,
		"startAtLogin": false,
		"disableHardwareAcceleration": false,
		"restartOnConfigChanges": false,
		"trayClickPlayPause": false,
		"autoResetAppCache": false,
		"resumeOnStart": true,
		"proxy": "",
		"singleInstanceLock": true
	},
	"plugins": {
		"navigation": {
			"enabled": false
		},
		"adblocker": {
			"enabled": false,
			"cache": true,
			"additionalBlockLists": []
		},
		"shortcuts": {
			"enabled": false,
			"overrideMediaKeys": false
		},
		"downloader": {
			"enabled": false,
			"ffmpegArgs": [],
			"preset": "mp3"
		},
		"last-fm": {
			"enabled": false,
			"api_root": "http://ws.audioscrobbler.com/2.0/",
			"api_key": "04d76faaac8726e60988e14c105d421a",
			"secret": "a5d2a36fdf64819290f6982481eaffa2"
		},
		"discord": {
			"enabled": false,
			"activityTimoutEnabled": true,
			"activityTimoutTime": 600000,
			"listenAlong": true
		},
		"notifications": {
			"enabled": false,
			"unpauseNotification": false,
			"urgency": "normal",
			"interactive": false
		},
		"precise-volume": {
			"enabled": false,
			"steps": 1,
			"arrowsShortcut": true,
			"globalShortcuts": {
				"volumeUp": "",
				"volumeDown": ""
			}
		},
		"sponsorblock": {
			"enabled": false,
			"apiURL": "https://sponsor.ajay.app",
			"categories": [
				"sponsor",
				"intro",
				"outro",
				"interaction",
				"selfpromo",
				"music_offtopic"
			]
		},
		"video-toggle": {
			"enabled": false,
			"forceHide": false,
			"hideVideo": true
		},
		"disable-autoplay": {
			"enabled": false
		},
		"audio-compressor": {
			"enabled": false
		}
	},
	"__internal__": {
		"migrations": {
			"version": "1.16.0"
		}
	},
	"window-position": {
		"x": 480,
		"y": 29
	},
	"window-maximized": true
}
```

The new:
```json
{
	"window-size": {
		"width": 1100,
		"height": 550
	},
	"url": "https://music.youtube.com",
	"options": {
		"tray": false,
		"appVisible": true,
		"autoUpdates": true,
		"hideMenu": false,
		"startAtLogin": false,
		"disableHardwareAcceleration": false,
		"restartOnConfigChanges": false,
		"trayClickPlayPause": false,
		"autoResetAppCache": false,
		"resumeOnStart": true,
		"proxy": ""
	},
	"plugins": {
		"navigation": {
			"enabled": true
		},
		"adblocker": {
			"enabled": true,
			"cache": true,
			"additionalBlockLists": []
		},
		"shortcuts": {
			"enabled": false,
			"overrideMediaKeys": false
		},
		"downloader": {
			"enabled": false,
			"ffmpegArgs": [],
			"preset": "mp3"
		},
		"last-fm": {
			"enabled": false,
			"api_root": "http://ws.audioscrobbler.com/2.0/",
			"api_key": "04d76faaac8726e60988e14c105d421a",
			"secret": "a5d2a36fdf64819290f6982481eaffa2"
		},
		"discord": {
			"enabled": false,
			"activityTimoutEnabled": true,
			"activityTimoutTime": 600000,
			"listenAlong": true
		},
		"notifications": {
			"enabled": false,
			"unpauseNotification": false,
			"urgency": "normal",
			"interactive": false
		},
		"precise-volume": {
			"enabled": false,
			"steps": 1,
			"arrowsShortcut": true,
			"globalShortcuts": {
				"volumeUp": "",
				"volumeDown": ""
			}
		},
		"sponsorblock": {
			"enabled": false,
			"apiURL": "https://sponsor.ajay.app",
			"categories": [
				"sponsor",
				"intro",
				"outro",
				"interaction",
				"selfpromo",
				"music_offtopic"
			]
		},
		"video-toggle": {
			"enabled": false,
			"forceHide": false
		}
	},
	"__internal__": {
		"migrations": {
			"version": "1.16.0"
		}
	}
}
```

Alas, if you are asking for the `package.json`, please tell me where to find it, but I am pretty sure I cannot give you the old one, since I have reinstalled everything again and again.

**Comment by Araxeus** [comment]
Yeah sorry I did mean `config.json`

anyway, the config seems to be valid, I personally have no idea what's going on here...

this might be fixed when 1.17.0 will be released (specifically because of the electron version bump)

**Comment by Diamondemon** [comment]
All right, I will be waiting, then :D
